english/play-with-dog/dog.md

```diff
@@ -1,12 +1,14 @@
 You walk down Valencia and see a little french bulldog appear around the corner.
 
-You follow her down 18th and end up at Dolores Park. You dodge around all of the 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.